### PR TITLE
Rewrite internal paper dropbox links

### LINKED ISSUE #9: Make sure internal links work
Links from one Playbook document to another Playbook document now point to Dropbox Paper rather than playbook.voorhoede.nl

For example on this page: https://playbook.voorhoede.nl/voorhoede-events/voorhoede-events.html

### PR DESCRIPTION
**Pull request by Siilwyn**
Closes #9

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
+};
+
 module.exports = {
   title: 'Playbook',
   themeConfig: {
-    sidebar: generateSidebar(JSON.parse(readFileSync('docs/dump.json'))),
+    sidebar: generateSidebar(docsMetaData),
+  },
+  extendMarkdown: markdown => {
+    markdown.use(markdownItForInline, 'internal-link', 'link_open', (tokens, idx) => {
```
Comment (jbmoelker) [inline docs/.vuepress/config.js]: make this a bit easier to understand?
`idx` -> `index`
`hi` -> ?
put `path.parse(hi)` into a meaningful variable as you use it multiple times?
(`const doc = path.parse(hi);` and `const dir = path.parse(doc.dir);`?)

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
+
+const docsMetaData = require('../dump.json');
 const generateSidebar = require('../../src/generate-sidebar.js');
 const manifest = require('./public/manifest.json');
 
+const derp = url => {
+  const urlMatch = url.match(/paper\.dropbox\.com\/doc\/.+--\S{26}-(\w{21})/);
+
+  if (urlMatch) {
+    const docMatch = docsMetaData.find(doc => doc.id === urlMatch[1]);
```
Comment (jbmoelker) [inline docs/.vuepress/config.js]: Make it easier to understand what comes out of the urlMatch?
```js
const docId = urlMatch[1];
const docMatch = docsMetaData.find(doc => doc.id === docId);
```

File: docs/.vuepress/config.js [hunk 3]
```diff
@@ -1,13 +1,41 @@
...
+    console.log(urlMatch, docMatch);
```
Comment (jbmoelker) [inline docs/.vuepress/config.js]: drop the log.

File: docs/.vuepress/config.js [hunk 4]
```diff
@@ -1,13 +1,52 @@
...
+} = require('sanctuary');
+
+const getDocumentLocation = pipe([
+    match(/paper\.dropbox\.com\/doc\/.+--\S{26}-(\w{21})/),
+    chain(match => match.groups[0]),
+    chain(urlId => find
+      (doc => doc.id === urlId)
+      (documentsMetaData)
+    ),
+    map(compose (path.parse) (prop('location'))),
```
[the remark this answers is not in the document]
Comment (Siilwyn) [reply docs/.vuepress/config.js]: Sanctuary has a convention of using spaces when 'currying': https://sanctuary.js.org/#currying.

I'm not sure how to use it in this case though, for now I only used it when a function takes multiple arguments like `compose`, the spacing is a bit inconsistent on line 41 though so I'll set that straight.

**Pull request merged**